package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "uploadtest",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@storybook/addon-actions": "^3.4.11",
+    "@storybook/react": "^3.4.11",
+    "bootstrap": "^4.1.3",
+    "enzyme": "^3.7.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "jquery": "^3.3.1",
+    "popper.js": "^1.14.4",
+    "prop-types": "15.6.2",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
+    "react-redux": "^5.1.0",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.0.5",
+    "redux": "^4.0.1",
+    "sass": "^1.14.3"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public",
+    "sass": "sass --watch src/sass/main.scss src/main.css"
+  },
+  "eslintConfig": {
```

Comment:
This field is redundant in `package.json` if we are using `.eslintrc.js`.

Reply:
Right, that makes sense, it is removed now.